i18next-parser.config.js

```diff
@@ -68,7 +68,7 @@ module.exports = {
   sort: false,
   // Whether or not to sort the catalog
 
-  useKeysAsDefaultValue: false,
+  useKeysAsDefaultValue: true,
```

Comment:
@kelvin-muchiri which one should we leave as the default, IMO `false` makes it easier to see which translations have not been added.